.changeset/every-walls-grow.md

```diff
@@ -3,6 +3,6 @@
 "fluid-framework": minor
 ---
 
-Detect arrayNode iterator invalidation
+fluid-framework, tree: Detect arrayNode iterator invalidation
```

Comment:
I'm not a big fan of this pattern. These package names are already called out in the metadata above. Requiring us to call them out here again seems redundant.

This is aimed at denoting the "main" affected packages, right? Could we accomplish this by introducing syntax for the metadata at the top to capture this instead?

Reply:
> I'm not a big fan of this pattern. These package names are already called out in the metadata above. Requiring us to call them out here again seems redundant.

I am not suggesting we manually do this moving forward. We do not have any code that creates these prefixes today, so it's all manual. If we like the layout, but not the implementation (manual), then we can add that to the list of features we want.

> This is aimed at denoting the "main" affected packages, right? Could we accomplish this by introducing syntax for the metadata at the top to capture this instead?

We could add more metadata, but my proposal is just to use the first package in the list as the "main" package. Again, we can automate stuff once we know what output we want.

Reply:
Why are we updating the title lines here then?

Reply:
> Why are we updating the title lines here then?

Maybe I am confused... We are doing a release on Monday, and we won't have time to (a) learn what we want to automate and (b) automate it before that release. So for the release we'll still be using the existing release note tools, which take the changeset contents as-is.

Reply:
Oh, so these title updates are part of an existing automated flow? Sorry, I thought we were doing manual tinkering with the existing changelogs to support some new pattern.

Reply:
Maybe the latest iteration will help clarify. The release notes are now included in this PR as they are produced by the tools we have. So you can see exactly what the automation produces and how the title changes make their way into the notes.